tests/mirelExampleTest.spec.js

```diff
@@ -0,0 +1,7 @@
+import { test, expect } from "@playwright/test";
+
+test("Verify that has the correct url", async ({ page }) => {
+  await page.goto("https://magento.softwaretestingboard.com/");
+
+  await expect(page).toHaveURL("https://magento.softwaretestingboard.com/");
```

Comment:
follow the instructions, toHaveTitle not toHaveUrl

Reply:
The extra example, as first task was to replicate what the pw example asked. And it asked to use the toHaveTitle method, not the toHaveUrl